dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops

```diff
@@ -63,6 +63,9 @@ CWMOT5007.tel.enabled.conflict=CWMOT5007W: Conflicting configuration for the ote
 CWMOT5007.tel.enabled.conflict.explanation: MicroProfile Telemetry can be enabled for the whole server by using system properties or environment variables. If it isn't enabled for the whole server, it can be enabled for an application by setting MicroProfile Config properties; for example, by setting variables in the server.xml file. This warning is given if Telemetry is enabled for the whole server, but the MicroProfile Config properties disable Telemetry for an application. Because Telemetry is enabled for the whole server, the configuration for that application is ignored.
 CWMOT5007.tel.enabled.conflict.useraction: Specify the settings to enable or disable OpenTelemetry instances by using either environment variables and system properties or other MP Config sources but not both.
 
+CWMOT5008.mptelemetry.unknown.access.log.field=CWMOT5008W: The MicroProfile Telemetry logging feature does not recognize the [ {0} ] telemetry access log format.
+CWMOT5008.mptelemetry.unknown.access.log.field.explanation=The Liberty runtime environment does not provide the specified log format.
+CWMOT5008.mptelemetry.unknown.access.log.field.useraction=Specify either default or logFormat for the mpTelemetry accessLogFields attribute.
```

Comment:
- what is the likely error here? Will setting accessLogFields resolve the problem? Or does the user need to use a supported log format?
- Would the user need to specify `default` in this case? If they are using an unsupported format and have not already defined accessLogFields, aren't they already declaring `default` format "by default"?

Reply:
If accessLogFields isn't configured but Telemetry accessLogs are enabled, we set the log format to default.
If a user configures an invalid log format in the accessLogFields attribute, we print out the warning and set it to "default" automatically. 

So we'll always set it to default if access logging is enabled for Telemetry unless logFormat is configured, we're just informing them that whatever they configured isn't valid.

Reply:
Also for your second point, " If they are using an unsupported format and have not already defined accessLogFields"

The unsupported formats we're referring to are for accessLogFields:

<mpTelemetry source="accessLog" accessLogFields="logFormat"/>
<mpTelemetry source="accessLog" accessLogFields="invalidInput"/> -> set to default automatically and print an error